### DISCUSSION TITLE
Still overthinking or have you already said goodbye to a have a discord community?

### DISCUSSION THREAD

**Discussion by onexbash** [zed-industries/zed#8076] [Feature Requests]
It seems like you're not really happy with using discord for zed, it leads to people posting issues/bugs & feature reuquests there while they actually belong in this gh repo. 

I agree that its not really target-oriented if the discord is used like this, but Discord is also a great place to get help from the community and connect the community. I always point at vercel's Next.js, because this discord was essential for me to during my rise of learning React & Next & builds connections and friendships with other devs.

So why not reconceptualize the concept of the discord server from a place where issues & bugs are posted(thats githubs task) to a place where questions and great conversation take place?

Especially for a new project like Zed, with many many people replacing vs code with it,
it would be so helpful to have a place where you can share and receive knowledge.

And I'm pretty sure there will be a lot of people that would love to support the project by helping as a discord mod / supporter / helper, whatever. Its win win win. 

Please dont just burry the whole idea :( Vercel proves that it works great if it's well thought out.

**Comment by JosephTLyons** [comment]
Hey @fschlegelone, I'm curious if you could provide a bullet-point list of the things you'd like to see changed about the Discord community. I'm also curious what might of led you to this conclusion:

> It seems like you're not really happy with using discord for zed

Genuinely wanting to learn how to do community better, any tips are greatly appreciated.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
